MaxText/input_pipeline.py

```diff
@@ -96,7 +96,8 @@ def preprocessing_pipeline(
   shift: bool = True,
   drop_remainder: bool = True,
   prefetch_size = tf.data.experimental.AUTOTUNE,
-  data_sharding = None
+  data_sharding = None,
+  seed = 0,
```

Comment:
I am also planning to add an initial random seed for the model weights, so we need to be more specific than "seed", can we call this "data_shuffle_seed" instead?